Feature/201 automatic setup qa engine in google cloud (#341)
devonfw/hangar: documentation/src/common_templates/setup-sonarqube-instance.asciidoc

```
terraform output $outputKeyName
```

NOTE:  Remember that command needs `terraform.tfstate` file to work.

==== Destroy SonarQube instance

As long as you keep the `terraform.tfstate` file generated when creating the SonarQube instance, you can easily destroy it and all associated resources by executing:

```
terraform destroy
```

==== Modify SonarQube instance infrastructure

As long as you keep the `terraform.tfstate` file generated when creating the SonarQube instance, you can apply changes to the infrastructure deployed.

If you are going to apply a change in the infrastructure, you will have to modify the terraform files and reapply the changes with the command `terraform apply`.

IMPORTANT: In windows, keep in mind that after applying any changes, you will lose the value of the token so be sure to copy or write it down before applying any changes. To avoid this we have implemented a method but to work you must store the standard terraform output in a file called terraform.tfoutput. This can be done with the following command:

```
terraform output > terraform.tfoutput
```

== Change Sonarqube default admin password

> After having deployed sonarqube by following this guide, you will be able to access SonarQube web interface on the url provided by terraform output and the following credentials:

Reviewer: ```suggestion
After a few minutes, you will be able to access SonarQube web interface on the public URL provided by Terraform output with the following credentials:
```

* Username:   `admin`
* Password:   `admin`

> IMPORTANT: Change the default password promptly. After that, update the password in terraform vars, you can do it manually or with the next command:
> 
> ```
> ./set-terraform-variables.sh --sonarqube_password ${YOUR_NEW_PASSWORD}
> ```

Reviewer: ```suggestion
IMPORTANT: Change the default password promptly. After that, update the password in Terraform configuration: `./set-terraform-variables.sh --sonarqube_password <new password>`.
```

> == Appendix: More information about terraform for {provider_name}

Reviewer: ```suggestion
== Appendix: More information about Terraform for {provider_name}
```

* {terraform_tutorials}[Official Terraform tutorials]
ifdef::extra_information[]

> {extra_information}

Reviewer: ```suggestion
* {extra_information}
```
